content/en/tracing/setup_overview/open_standards/otlp_ingest_in_the_agent.md

```diff
@@ -0,0 +1,233 @@
+---
+title: OTLP Trace Ingestion by the Datadog Agent
+kind: documentation
+description: 'Ingest OTLP trace data through the Datadog Agent'
+further_reading:
+aliases:
+---
+
+
+OTLP Ingest in the Agent is a way to send telemetry data directly from [OpenTelemetry SDKs][1] to Datadog Agent. Since versions 6.32.0 and 7.32.0, the Datadog Agent can ingest OTLP traces and [OTLP metrics][2] through gRPC or HTTP.
+
+OTLP Ingest in the Agent allows you to use trace observability features in the Datadog Agent. Because the application is instrumented with OpenTelemetry SDK, some Datadog Tracing Library specific features aren't available for the ingested data including Application Security, Continuous Profiler, Runtime Metrics, and Ingestion Rules.
+
+<div class="alert alert-warning">OpenTelemetry Metrics ingestion is in beta and its behavior and configuration may change.</div>
```

Comment:
I think OTLP here is slightly more accurate

```suggestion
<div class="alert alert-warning">OTLP Metrics ingestion is in beta and its behavior and configuration may change.</div>
```

Also, could we add the same banner on https://docs.datadoghq.com/metrics/otlp/ for consistency?